analytics_data_api/v0/connections.py

```diff
@@ -0,0 +1,69 @@
+import json
+import time
+
+from boto.connection import AWSAuthConnection
+from django.conf import settings
+from elasticsearch import Connection
+
+
+class BotoHttpConnection(Connection):
+    """
+    Uses AWS configured connection to sign requests before they're sent to
+    elasticsearch nodes.
+    """
+
+    connection = None
+
+    def __init__(self, host='localhost', port=443, **kwargs):
```

Comment:
AWS elasticsearch service exposes the service on 443 so that you can use HTTPS URLs without specifying a port:

https://search.lkueroi73897.aws.com/

Instead of:

https://search.lkueroi73897.aws.com:9200/

Given that you would only use this connection class for aws elasticsearch clusters, I defaulted the port to 443.